### PR TITLE
Add docs for recovering CELO from an ETH wallet

### PR DESCRIPTION
**Pull request by henry-zhang-bohan**
Moving documentation from here: https://docs.google.com/document/d/1ivSdDDDequdblNM66q5-a58rnt5Ug4AzHPpqRrXv0ng/edit?usp=sharing

### CODE REVIEW

File: packages/docs/celo-holder-guide/eth-recovery.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Prerequisites
+
+This guide assumes that you have access to the following:
+
+- A computer with a [Command Line Interface](https://en.wikipedia.org/wiki/Command-line_interface). You can access it following these instructions:
+    - [Windows](https://www.howtogeek.com/270810/how-to-quickly-launch-a-bash-shell-from-windows-10s-file-explorer/)
+    - [Mac](https://macpaw.com/how-to/use-terminal-on-mac)
+    - [Linux](https://askubuntu.com/questions/196212/how-do-you-open-a-command-line)
+- The [24-word recovery phrase](https://kb.myetherwallet.com/en/security-and-privacy/what-is-a-mnemonic-phrase/) of your Ethereum address
```
Comment (marekolszewski) [inline packages/docs/celo-holder-guide/eth-recovery.md:14]: Should we add a requirement around having the celocli installed?

File: packages/docs/celo-holder-guide/eth-recovery.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+3. Replace the `<word>`s in brackets with the 24 words from your recovery phrase
+4. Press ctrl-o to save
+5. Press ctrl-x to exit
+
+### Recover your Ethereum address on Celo
+
+Recover your Ethereum address on the Celo network:
+
+```
+celocli account:new --indexAddress 0 --mnemonicPath recovery.txt --derivationPath "m/44'/60'/0'/0" --node https://rc1-forno.celo-testnet.org
```
Comment (marekolszewski) [inline packages/docs/celo-holder-guide/eth-recovery.md:40]: Do we have a better URL for mainnet forno yet? If so, let's use it here.

File: packages/docs/celo-holder-guide/eth-recovery.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+Replace `<accountAddress>` with the `accountAddress` you got from the previous step.
+
+### Transfer CELO
+
+Now, you can transfer your CELO to an address of choice:
+
+```
+celocli transfer:gold --from <accountAddress> --to <addressOfChoice> --value <valueInCeloWei> --privateKey <privateKey> --node https://rc1-forno.celo-testnet.org
```
Comment (marekolszewski) [inline packages/docs/celo-holder-guide/eth-recovery.md]: Can we use transfer:celo here now that it's supported?

**Review by marekolszewski — CHANGES_REQUESTED** [review]
Looks great! Left some suggestions.

File: packages/docs/celo-holder-guide/eth-recovery.md (hunk 2 above)
Comment (henry-zhang-bohan) [reply packages/docs/celo-holder-guide/eth-recovery.md:40]: Still not implemented yet :(

File: packages/docs/celo-holder-guide/eth-recovery.md (hunk 1 above)
Comment (henry-zhang-bohan) [reply packages/docs/celo-holder-guide/eth-recovery.md:14]: added

**Review by marekolszewski — APPROVED** [review]
Looks great!

**Pull request merged**